packages/api/src/task/scheduler.ts

```diff
@@ -592,7 +592,7 @@ export class TaskScheduler {
     filters?: { allowedPhases: Array<Asset["status"]["phase"]> },
   ) {
     if (typeof asset === "string") {
-      asset = await db.asset.get(asset);
+      asset = await db.asset.get(asset, { useReplica: false });
```

Comment:
Reminder that what we really need to investigate is how the replication lag is getting so high. It is not normal for the db to be so out of date that _minutes_ after creating the asset/task it is still not in the replicas.

That said, if we're going the "drop the replica" approach, we should probably update every call to the DB in this file to use the primary instead. Can just search for `db.` for all calls to the `db` and add the replica option